scripts/flail.py

```diff
@@ -126,7 +126,11 @@ def obtain_var_name(self):
                                       .replace('.goto', '_buf')
                                       .replace('.txt', '_buf')
                                       .replace('buildidobj', 'buildidstring')
-                                      .replace('-', '_'))
+                                      .replace('-', '_')
+                                      # CPP THINGS
+                                      .replace('vector', '__esbmc_om_vector') 
+                                      .replace('new', '__esbmc_om_new')
+                                      .replace('ctime', '__esbmc_om_ctime'))
```

Comment:
Could you explain why these are necessary?

Reply:
It isn't. I didn't notice the PREFIX bit of your previous comment